modules/search/class-jetpack-instant-search.php

```diff
@@ -115,7 +115,15 @@ protected function inject_javascript_options() {
 			);
 		}
 
-		$prefix  = Jetpack_Search_Options::OPTION_PREFIX;
+		$prefix         = Jetpack_Search_Options::OPTION_PREFIX;
+		$posts_per_page = (int) get_option( 'posts_per_page' );
+		if ( $posts_per_page > 20 ) {
```

Comment:
Hey @gibrown any reason we don't want to right this with an OR statement as we are defaulting to the same 20 posts max? it would give us 1 line to change vs 2  if the api limit changes.

if( $posts_per_page > 20 || $posts_per_page <= 0 ) {

Reply:
ya good point